airflow-core/docs/administration-and-deployment/plugins.rst

```diff
@@ -20,6 +20,10 @@
 Plugins
 ========
 
+.. warning::
+
+    As of Airflow version 3.0.0, support for user-defined macros is pending.
```

Comment:
```suggestion
    As part of Airflow 3.0.0 development, support for user-defined macros is pending. This will be added soon in one of the 3.0.0 patch version or 3.1
```

Reply:
I think it's best to just link to the Github issue that track this